medusa/server/api/v1/core.py

```diff
@@ -1965,8 +1965,8 @@ def __init__(self, args, kwargs):
         self.initial, args = self.check_params(args, kwargs, 'initial', None, False, 'list', list(quality_map))
         self.archive, args = self.check_params(args, kwargs, 'archive', None, False, 'list',
                                                list(quality_map).remove('unknown'))
-        self.flatten_folders, args = self.check_params(args, kwargs, 'flatten_folders',
-                                                       bool(app.FLATTEN_FOLDERS_DEFAULT), False, 'bool', [])
+        self.season_folders, args = self.check_params(args, kwargs, 'season_folders',
+                                                       bool(not app.FLATTEN_FOLDERS_DEFAULT), True, 'bool', [])
```

Comment:
I suggest [bumping the API version up](https://github.com/pymedusa/Medusa/blob/feature/rename-flatten-folders/medusa/server/api/v1/core.py#L101) due to these changes.

Reply:
Also,